packages/widgets/src/components/PieChart/PieChartLegend.tsx

```diff
@@ -12,40 +12,63 @@ export const PieChartLegend = ({
   renderLabel,
   renderValue,
   markerStyles,
+  currency,
   labelListStyles,
   valueListStyles,
+  variant = 'valueList',
 }: PieChartLegendProps) => {
   const theme = useTheme();
+  const isValueList = variant === 'valueList';
   return (
     <Fragment>
       <PieChartLegendList css={labelListStyles}>
         {data.map((item, index) => {
-          const { id, label } = item;
+          const { id, label, value, legendValue } = item;
           return (
             <li key={`tag-${id}`}>
               <PieChartLegendMarker
                 color={colors[index] || 'purple'}
                 as={'span'}
                 css={markerStyles}
               />
-              <Typography variant="h6">
-                {typeof renderLabel === 'function' ? renderLabel(item) : label}
-              </Typography>
+              {isValueList ? (
+                <Typography variant="h6">
+                  {typeof renderLabel === 'function'
+                    ? renderLabel(item)
+                    : label}
+                </Typography>
+              ) : (
+                <Typography variant="subtitle" css={{ lineHeight: '12px' }}>
+                  <p>
+                    {legendValue as React.ReactNode} {label}
+                  </p>
+                  <p css={{ fontWeight: '400' }}>
+                    {value} {currency}
+                  </p>
+                </Typography>
+              )}
             </li>
           );
         })}
       </PieChartLegendList>
-      <PieChartLegendList css={valueListStyles}>
-        {data.map((item) => (
-          <li key={`subtitle-${item.id}`}>
-            <Typography variant="subtitle" color={theme.colors.greyDarker60}>
-              {typeof renderValue === 'function'
-                ? renderValue(item)
-                : item.value}
-            </Typography>
-          </li>
-        ))}
-      </PieChartLegendList>
+      {isValueList && (
+        <PieChartLegendList css={valueListStyles}>
+          {data.map(
+            (item) =>
+              !item.customLegendItem && (
```

Comment:
Is this okay comparing both `isValueList` and `item.customLegentItem`? In case, `isValueList` variable is true and `item.customLegendItem` is also true, we output an empty `ul` tag

Reply:
Forgot to remove. Done